examples/fast_neural_style/neural_style.py

```diff
@@ -0,0 +1,249 @@
+                                  help="path to style-image")
+    train_arg_parser.add_argument("--test_image", type=str, default="test",
+                                  help="path to test-image")
+    train_arg_parser.add_argument("--checkpoint_model_dir", type=str, default='/tmp/checkpoints',
+                                  help="path to folder where checkpoints of trained models will be saved")
+    train_arg_parser.add_argument("--checkpoint_interval", type=int, default=1,
+                                  help="number of batches after which a checkpoint of trained model will be created")
+    train_arg_parser.add_argument("--stylized_test_dir", type=str, default='/tmp/images/stylized_test',
+                                  help="path to folder where stylized test image will be saved at end of each epoch")
+    train_arg_parser.add_argument("--image_size", type=int, default=256,
+                                  help="size of training images, default is 256 X 256")
+    train_arg_parser.add_argument("--style_size", type=int, default=None,
+                                  help="size of style-image, default is the original size of style image")
+    train_arg_parser.add_argument("--cuda", type=int, default=1,
+                                  help="set it to 1 for running on GPU, 0 for CPU")
+    train_arg_parser.add_argument("--seed", type=int, default=42,
+                                  help="random seed for training")
+    train_arg_parser.add_argument("--content_weight", type=float, default=1e5,
+                                  help="weight for content-loss, default is 1e5")
+    train_arg_parser.add_argument("--style_weight", type=float, default=1e10,
+                                  help="weight for style-loss, default is 1e10")
+    train_arg_parser.add_argument("--lr", type=float, default=1e-3,
+                                  help="learning rate, default is 1e-3")
+
+    eval_arg_parser = subparsers.add_parser("eval", help="parser for evaluation/stylizing arguments")
+    eval_arg_parser.add_argument("--content_image", type=str, required=True,
+                                 help="path to content image you want to stylize")
+    eval_arg_parser.add_argument("--content_scale", type=float, default=None,
+                                 help="factor for scaling down the content image")
+    eval_arg_parser.add_argument("--output_image", type=str, required=True,
+                                 help="path for saving the output image")
+    eval_arg_parser.add_argument("--model", type=str, required=True,
+                                 help="saved model to be used for stylizing the image.")
+    eval_arg_parser.add_argument("--cuda", type=int, required=True,
+                                 help="set it to 1 for running on GPU, 0 for CPU")
+
+    args = main_arg_parser.parse_args()
+
+    if args.subcommand is None:
+        print("ERROR: specify either train or eval")
```

Comment:
these should be changed to `raise ValueError` instead of `print` and `sys.exit`

Reply:
Incorporated